ui/shared/nft/NftMedia.tsx

```diff
@@ -9,24 +9,26 @@ import useFetch from 'lib/hooks/useFetch';
 
 import NftFallback from './NftFallback';
 import NftHtml from './NftHtml';
+import NftHtmlWithFullscreen from './NftHtmlWithFullscreen';
 import NftImage from './NftImage';
+import NftImageWithFullscreen from './NftImageWithFullscreen';
 import NftVideo from './NftVideo';
+import NftVideoWithFullscreen from './NftVideoWithFullscreen';
```

Comment:
Do we really need 3 separate components for this? I think it is better to configure the NftMedia that is in the fullscreen modal by simply passing the necessary styles to it. Now, we have a lot of duplication in media components.

Reply:
we need. you know I'm not a fan of sophisticated abstractions. I can move common props to utils, but not a unite component

Reply:
I assume that passing style props to already existing components is not something sophisticated.

What I mean is apart from the "content" variable here, we can have a "modal" variable that will render the same media components but with custom style props:

```js
const modal = (() => {
    switch (type) {
      case 'video':
        return (
          <NftMediaFullscreenModal>
            <NftVideo { ...styleProps }/>
          </NftMediaFullscreenModal>
        );
     etc...
    }
})();
```

It is just reusing the existing components with different props. I don't get why it is over-complicated.

Reply:
I don't see what would be better in that case, honestly.

Reply:
it doesn't work for the html type, fullscreen and small has different structures, so it's not "the same media component"